atlassian_jwt_auth/__init__.py

```diff
@@ -13,3 +15,7 @@
 from atlassian_jwt_auth.verifier import (
     JWTAuthVerifier,
 )
+
+if sys.version_info >= (3, 5):
```

Comment:
Unfortunately without this check nose tries to import it in py2.7 tests and fails with SyntaxError